libraries/Microsoft.Bot.Connector/Authentication/BuiltinBotFrameworkAuthentication.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.Logging;
+using Microsoft.Rest;
+
+namespace Microsoft.Bot.Connector.Authentication
+{
+    internal abstract class BuiltinBotFrameworkAuthentication : BotFrameworkAuthentication
```

Comment:
Not a big deal, but personally I'd prefer 'Default' to 'BuiltIn'

Reply:
Firstly this is purely internal; and secondly this is temporary (shippable but ultimately temporary), this builtin code uses the current built in support for Public and Gov. The default (i.e. no configuration is public) but the future implementation of that default might (hopefully) be parameterizing the Parameterized class with the constants for the public cloud.

Anyhow "built in" and "default" are slightly different concepts.

Reply:
okay, thanks for the explanation @johnataylor